config/checkstyle/checkstyle.xml

```diff
@@ -93,12 +86,14 @@
                   value="Do not use @author. Changes and authors are tracked by the content manager." />
     </module>
 
+    <module name="SuppressWarningsFilter" />
```

Comment:
Use the provided suppressions below, don't add your own:
```
// CHECKSTYLE: <RULE_NAME> OFF
// CHECKSTYLE: <RULE_NAME> ON
```

Reply:
Fixed in 129fbb3.